### PR TITLE
Updated old constructor formats in Inheritance section.

### PR DESCRIPTION
**Pull request by MatthewDLudwig**
As requested by /u/chriseth I have changed the issues with the old constructor format still being present in the Inheritance section.  If there are any further places you know of with the old format, feel free to let me know and I can fix them in my branch and delay this pull request until they've all been fixed.

### CODE REVIEW

File: docs/contracts.rst [hunk 1]
```diff
@@ -844,7 +844,7 @@ Details are given in the following example.
     pragma solidity ^0.4.16;
```
Comment (axic) [inline docs/contracts.rst]: Need to update the version here to `^0.4.22`.

File: docs/contracts.rst [hunk 2]
```diff
@@ -916,7 +916,7 @@ seen in the following example::
     pragma solidity ^0.4.0;
```
Comment (axic) [inline docs/contracts.rst]: Need to update here too (same version as a above).

File: docs/contracts.rst [hunk 3]
```diff
@@ -945,7 +945,7 @@ derived override, but this function will bypass
...
```
Comment (axic) [inline docs/contracts.rst]: Need to update here too (same version as a above).

**Review by axic — CHANGES_REQUESTED** [review]

**Comment by axic** [comment]
Can you please squash those changes into a single commit?

**Comment by MatthewDLudwig** [comment]
Closing pull request in order to create a better, more organized one.  Will need to practice squashing as I seem to have only created more commits.

**Pull request closed without merging**